mrp_production_type/views/mrp_production_views.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="mrp_production_form_view" model="ir.ui.view">
+        <field name="name">mrp.production.form</field>
+        <field name="model">mrp.production</field>
+        <field name="inherit_id" ref="mrp.mrp_production_form_view" />
+        <field name="arch" type="xml">
+            <field name="bom_id" position="after">
+                <field name="mo_type_id" required="True" />
```

Comment:
```suggestion
                <field name="mo_type_id" required="1" />
```

Reply:
required="True" is not wrong. But most of code is using "1", so it will be easier if we want to find it in the future.